src/general/MainMenu.cs

```diff
@@ -234,11 +221,11 @@ public override void _Process(float delta)
                     GD.Print("We are most likely a store version of Thrive, showing the thanks dialog");
 
                     // The text has a store link template, so we need to update the right links into it
-                    thanksDialogText.ExtendedBbcode =
+                    thanksDialog.DialogText =
```

Comment:
I got this far and realized why the use of rich text in the dialog is needed. Still the centering should just set the text alignment property, surely that's still better than extra complicating the markup handling?

Reply:
I would if I could but there's no alignment property in RTL, everything is done through bbcode.